tt_eager/tt_dnn/op_library/bmm/kernels/dataflow/reader_bmm_tile_layout_in0_sender_receiver_padding_block_sharded.cpp

```diff
@@ -32,12 +36,16 @@ void kernel_main() {
     volatile tt_l1_ptr uint32_t * in0_mcast_noc_y          = (volatile tt_l1_ptr uint32_t*)(get_arg_addr(5 + num_x));
 
     constexpr uint32_t cb_id_in0 = 0;
-    constexpr uint32_t cb_id_in2 = 2; // Sharded cb
-
+    constexpr uint32_t cb_id_in2 = 2;  // Sharded cb
+    constexpr uint32_t cb_id_in4 = 4; // Sharded cb
```

Comment:
i think it's possible to directly copy to cb in0 and mcast that cb to other cores. Perf-wise it should be the same as long as in0 is double buffered.